dts/arm/96b_carbon.fixup

```diff
@@ -25,3 +25,4 @@
 #define CONFIG_I2C_1_NAME			ST_STM32_I2C_V1_40005400_LABEL
 #define CONFIG_I2C_1_EVENT_IRQ			ST_STM32_I2C_V1_40005400_EVENT
 #define CONFIG_I2C_1_ERROR_IRQ			ST_STM32_I2C_V1_40005400_ERROR
+#define CONFIG_I2C_1_DEFAULT_CFG		(ST_STM32_I2C_V1_40005400_ZEPHYR_SPEED << 1) | I2C_MODE_MASTER
```

Comment:
Defining Master Mode here is not clean. Even as a temporary solution until we have a proper master/slave support.